.golangci.yml

```
@@ -0,0 +1,110 @@
+        - octalLiteral
+        - whyNoLint
+    
+    funlen:
+      lines: 100
+      statements: 50
+
+formatters:
+  enable:
+    - gofmt
+    - goimports
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)